go.mod

```
@@ -0,0 +1,59 @@
+module github.com/percona/pmm-managed
+
+go 1.16
+
+// Use for local development, but do not commit:
+// replace github.com/percona-platform/platform => /local-path/platform
```

```suggestion
// replace github.com/percona-platform/saas => ../saas
```
I assume it's a common case when all percona repos located in the same folder. So better to have `../saas` as replace target. In 95% cases developer should just uncomment this line

---

but saas is in percona platform, so maybe then `../../saas`

---

My modules projects located outside gopath, just in folder `~/projects/{pmm-managed,saas,pmm-agent}`.
I thought it's a more or less common approach. Now modules are enabled by default, so all modules projects work in Gopath without setting `GO111MODULES` variable, so I'm not sure what option is better. But at least this `replace` pattern already used in platform repos.